### ISSUE TITLE
Composite actions support?

### ISSUE THREAD

**Issue by druskus20** [rhysd/actionlint#401]
Would it be possible for this tool to work with composite actions too? 
Composite actions have turned out to be very useful for me to re-use functionality across repositories, but this action does not seem to support them. 
Most of the pitfalls of workflows / re-usable workflows still apply there.

Thanks!

**Comment by alexanderbazhenoff** [comment]
I agree, or please add ignoring of files list from config ASAP. Currently I should disable all linting in Super-Linter via enviornment variable: `FILTER_REGEX_EXCLUDE: .github/workflows/composite-actions/action.yml`. There will be no yaml lint either.

**Comment by MPV** [comment]
Is this a duplicate of...?
- https://github.com/rhysd/actionlint/issues/46

**Comment by make-github-pseudonymous-again** [comment]
> I agree, or please add ignoring of files list from config ASAP. Currently I should disable the entire linting of file in Super-Linter via enviornment variable: `FILTER_REGEX_EXCLUDE: .github/workflows/composite-actions/action.yml`. There will be no yaml lint either.

@alexanderbazhenoff You can also move your actions outside `.github/workflows`, e.g. to `.github/actions`.

**Comment by druskus20** [comment]
> Is this a duplicate of...?
> 
> * [Feature Request: Support linting action metadata files #46](https://github.com/rhysd/actionlint/issues/46)

I guess so, I was not able to find that issue when I originally posted. 
If it is, we can close this one.
